client/pokemon/pokemon-icon.view.html

```diff
@@ -0,0 +1,4 @@
+<div class="icon-container" ng-init="pokemon.parseCardViewProps()" flex
+  ng-class="['spr-box', pokemon.spriteClass, {'pokemon-egg': pokemon.data.isEgg}]" >
+  <span ng-if="::pokemon.data.isEgg" class="spr-box spr-box-egg"></span>
```

Comment:
Have you tested the `isEgg` logic here? I don't think the `isEgg` property will be returned from the server, because it's not in `POKEMON_FIELDS_FOR_CARD`. So the egg icon will never appear.